3ds Max/Max2Babylon/Max2Babylon_Package.py

```diff
@@ -1,14 +1,137 @@
+# BEFORE USING THIS SCRIPT, PLEASE MAKE SURE PYTHON AND THE PACKAGES LISTED IN THE README.MD ARE INSTALLED.
+
+# ---------- IMPORTS ----------
 import os
 import zipfile
 import shutil
+import json     # Changelog
+import requests  # Changelog
+import datetime  # Changelog
 
 versions = ['2015', '2017', '2018', '2019']
 projectFolderPrefix = './'
 buildPathPrefix = '/bin/Release/'
 
 packageFolderPrefix = '../'

```

Comment:
This and the project folder prefixes will have to be changed to align with the source code pathing used in L25

Reply:
Yup. I originally wrote a script on another folder, then moved the code when I realized that another script was there.